[1/n] Remove references to TorchScript in PyTorch docs

yiming0416 opened the pull request:
Summary: Removed jit_language_reference_v2.md

Test Plan:
CI

Rollback Plan:

Differential Revision: D78308009

svekars reviewed (approved):
LGTM, please check that it looks good in preview.

yiming0416 committed:
[1/n] Remove references to TorchScript in PyTorch docs (#158305)

Summary:

Removed jit_language_reference_v2.md

Test Plan:
CI

Rollback Plan:

Differential Revision: D78308009
